_posts/multiworld/2024-03-10-multiworld-with-nintendont.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld"
+tags: multiworld prime1 prime2
+---
+# Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld #  
+While playing Metroid Prime or Metroid Prime 2: Echoes Multiworld Randomizer on Dolphin Emulator is the most popular method, you can also play it on a real Wii or Wii U (Wii Mode) with a [custom fork of Nintendont](https://github.com/randovania/Nintendont/) with support for networked multiworld and automatic item tracking.  
+Using a real Wii and Wii U console may be favorable if your computer does not run emulated GameCube games well.  
+  
+## Requirements ##  
+  
+* **A Computer with an SD Card Reader**  
+  This guide assumes using Windows, but this should be similar for other operating systems.  
+* **All the prerequisites to use and generate a game with [Randovania](https://randovania.org/)**  
+* **An SD Card**  
+  Recommended format: FAT32 with 32 KB clusters  
+  * **⚠️ WARNING** Formatting will erase all the data on that card. Backup/copy your card's files to a different location before formatting.  
+  * exFAT can work, but can cause issues with other homebrew apps. NTFS will not work.  
+  * USB Storage may also work, but it's difficult to tell which models are reliable with the Wii and Wii U.  
```

Comment:
in general, the best external storage to use is a compatible USB hard drive. SD cards are the next best, and USB flash drives are the worst option